### DISCUSSION TITLE
2023 06 June Meetings thread

### DISCUSSION THREAD

**Discussion by kerneltime** [apache/ozone#4832] [Community Sync Meetings]
# June 5th
1. Scrubber design review
2. Pending PRs and Jiras
https://cloudera.zoom.us/rec/share/ZSTR24N4ATlQFvK9UK9xm9-v98TrK4a_BckMjen4FhJok4mnDNTl1QNgFAM-Er7J.rzIoYLG6pOeMl5Iw 
Passcode: v##8SHM0

**Comment by jojochuang** [comment]
June 8th APAC
* RATIS-1847 Ratis streaming memory leak.
* Qihoo reported their custom Ozone where OM’s backend uses Cassandra instead of Ratis HA.
They were able to achieve 20k txns per second. And their S3g read from Cassandra directly. Bypass OM.
* Ozone release planning; Ratis release planning.

**Reply by jojochuang** [reply]
June 15th APAC

* Ozone Manager performance improvements: https://medium.com/engineering-cloudera/ozone-performance-improvements-c0bb8a76dc94
* Hue file browser support for Ozone: https://gethue.com/blog/discover-the-power-of-apache-ozone-using-the-hue-file-browser/
* CommunityOverCode Asia CfP closing this weekend: https://apachecon.com/acasia2023/cfp.html
* DiDi is planning to upgrade to the latest. 1.4.0 release discussion. Current list of issues: https://docs.google.com/document/d/1g1h-63fvA-be-clvyVRAHLWehoadCnjNRWyX8Bp-UIU/edit

**Reply by jojochuang** [reply]
@ChenSammi i got distracted when Ivan was talking. If it's useful can you add some more?

**Comment by michelsumbul** [comment]
@jojochuang Do you have more info on the customize Ozone using Cassandra I will be very interested to know more, why and how

**Reply by jojochuang** [reply]
I'm only focusing on supporting HBase right now.

Basically HBase has dependency on HDFS and we're trying to replace HDFS with Ozone. For Cassandra, I am not how they integrate. But if Cassandra uses Hadoop Compatible File System API then it should be able to use Ozone by adding Ozone client jars.

**Comment by umamaheswararao** [comment]
2023 June 12th Community Sync Meeting Recording ( 9AM PST ):

https://cloudera.zoom.us/rec/share/ykD3YJe3nAla8PBdNIW6Ffjqx5yEA06vSRPmTa0yAnP3_a2FD6hOx9SotGVKhe4m.4A_ErQLE-Zvja28K
Passcode: 1gyd5H9#

**Comment by umamaheswararao** [comment]
2023 June 19th Community Sync Meeting Recording ( 9AM PST ):

https://cloudera.zoom.us/rec/share/_Nm8pWi6KCY0fdCG_G8gEnd-2kF-CkSdU76wINigMehGIc59JWICnT838F_zBF10.y85f6_J4-RuKQd9a
Passcode: 8!s0R=p+

**Comment by umamaheswararao** [comment]
2023 June 26th Community Sync Meeting Recording ( 9AM PST ):

https://cloudera.zoom.us/rec/share/Q0yE8tBH15YtnzIYm7QvM_rK3uyVPdjBs87ohv-xVCqyWhsWKjTWaI0Acemb0E2s.tUmT6nu04-5RRYwJ
Passcode: HB*SA!g3
